src/extension.ts

```diff
@@ -63,14 +63,15 @@ import { CredentialsStore } from './credentials/credentialsStore'
 
 let localize: nls.LocalizeFunc
 
-export async function activate(context: vscode.ExtensionContext) {
+export async function activate(context: vscode.ExtensionContext): Promise<any> {
```

Comment:
Definitely return a defined shape. The value returned here is [usable by any other VS Code extension](https://code.visualstudio.com/api/references/vscode-api#Extension%3CT%3E).

Reply:
why does that matter? The goal is not to expose this api, we haven't documented it. It is "hidden" and not intended to be used by other extensions.